monitoring/google/cloud/monitoring_v3/proto/uptime.proto

```diff
@@ -201,16 +292,10 @@ message UptimeCheckConfig {
   // regions.
   repeated UptimeCheckRegion selected_regions = 10;
 
-  // If this is true, then checks are made only from the 'internal_checkers'.
-  // If it is false, then checks are made only from the 'selected_regions'.
-  // It is an error to provide 'selected_regions' when is_internal is true,
-  // or to provide 'internal_checkers' when is_internal is false.
-  bool is_internal = 15;
-
```

Comment:
It looks like this is still being removed rather than deprecated.  Please do not merge.